onelogin/onelogin.saml.properties

```diff
@@ -0,0 +1,62 @@
+onelogin.saml2.strict = true
+onelogin.saml2.debug =  false
+
+###
+# Service Provider settings
+onelogin.saml2.sp.entityid = ${saml.sp.entity.id}
+onelogin.saml2.sp.assertion_consumer_service.url = ${saml.sp.acs.url}
+onelogin.saml2.sp.assertion_consumer_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST
+onelogin.saml2.sp.single_logout_service.url =
+onelogin.saml2.sp.single_logout_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect
+onelogin.saml2.sp.nameidformat = urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified
+onelogin.saml2.sp.x509cert =
+onelogin.saml2.sp.privatekey =
+
+###
+# Identity Provider settings
+onelogin.saml2.idp.entityid = ${saml.idp.entity.id}
+onelogin.saml2.idp.single_sign_on_service.url = ${saml.idp.sso.url}
+onelogin.saml2.idp.single_sign_on_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect
+onelogin.saml2.idp.single_logout_service.url =
+onelogin.saml2.idp.single_logout_service.response.url =
+onelogin.saml2.idp.single_logout_service.binding = urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect
+onelogin.saml2.idp.x509cert =
+# onelogin.saml2.idp.certfingerprint =
+# onelogin.saml2.idp.certfingerprint_algorithm = sha1
+
+###
+# Security settings
+onelogin.saml2.security.nameid_encrypted = false
+onelogin.saml2.security.authnrequest_signed = true
+onelogin.saml2.security.logoutrequest_signed = false
+onelogin.saml2.security.logoutresponse_signed = false
+onelogin.saml2.security.want_messages_signed =
+onelogin.saml2.security.want_assertions_signed = true
+onelogin.saml2.security.sign_metadata =
+onelogin.saml2.security.want_assertions_encrypted = ${saml.encryption.assertions.enabled}
+onelogin.saml2.security.want_nameid_encrypted = ${saml.encryption.nameid.enabled}
+onelogin.saml2.security.requested_authncontext = urn:oasis:names:tc:SAML:2.0:ac:classes:Password
+# todo should we enforce https password protection here?
```

Comment:
should we enforce https password protection here?
If we do, we may need to enable https on the Mock Idp, or make this a paramaterised property

Reply:
As discussed, I will test this out with my local version of the OneLogin java-saml Toolkit Sample Webapp